src/client/containers/ProfilePage/AdminProfile/AdminProfile.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import admin from '../../../assets/images/admin.png';
+import { Layout } from '../../../components';
+import { AppHeader } from '../../../components/Appheader/AppHeader.component';
+import { Avatar } from '../../../components/Avatar/Avatar';
+import Footer from '../../../components/footer/footer';
+import Header from '../../../components/Heading/Heading';
+import Input from '../../../components/Input/Input';
+import { useUser } from '../../../context/userContext';
+
+export const AdminProfilePage = () => {
+  const {
+    user: { name },
+  } = useUser();
+  return (
```

Comment:
Don't you want to fetch the data from that GET endpoint below?

Reply:
I just want to, is it possible to get using userContext, because user name is coming from user context.  It's just an idea, may be it is irrelevant

Reply:
Not sure what your question is here.

Reply:
Admin name is coming from user context, similarly can we get adminRole from user context?

Reply:
No, you use the endpoint you built for that.